docs/installation.md

```diff
@@ -0,0 +1,8 @@
+---
+id: installation
+title: Installation
+sidebar_label: Introduction
+---
+
+This is a link to [another document.](doc3.md)  
+This is a link to an [external page.](http://www.example.com)
```

Comment:
Let's remove these empty pages rather than have this kind of content please, it will make it easier to spot the missing parts of the docs.